inspire_dojson/common/rules.py

```diff
@@ -322,7 +323,7 @@ def legacy_creation_date(self, key, value):
     if 'legacy_creation_date' in self:
         return self['legacy_creation_date']
 
-    return value.get('x')
+    return normalize_date(value.get('x'))
```

Comment:
this doesn't work with `2017-10-00`/`2017-00-00`. You need to strip trailing `-00` first.

Reply:
We have those in `961__x`? Can I have an example?

Reply:
oh, sorry, this is `961__x`. I saw the `x` and thought about `111__x` in conferences.

Reply:
https://inspirehep.net/record/1364816

Reply:
Ah, yes, sure, but that is handled in this other part of the code: https://github.com/inspirehep/inspire-dojson/blob/41b9fdd8ab91e8c3a49b8d2472578f8b9b464f3b/inspire_dojson/conferences/rules.py#L36-L46